### DISCUSSION TITLE
on premise: Failed to create database

### DISCUSSION THREAD

**Discussion by kerkhc** [kimai/kimai#4700] [On-Premise: Installation & Updates] [answered]
### Describe the issue

After reading all the documentation and performing the installation steps I'm blocked at `bin/console kimai:install -n`
Error says:
`[ERROR] Failed to create database: An exception occurred in the driver: could not find driver`
Please give advice on how to solve this.
Linux Debian Bookworm, running as container on proxmox.

### I already tried

- [X] I've read and searched [the documentation](https://www.kimai.org/documentation/).
- [X] I've searched for similar issues in this repository.
- [X] I've searched for similar issues in [the discussions](https://github.com/kimai/kimai/discussions).

### Kimai version

2.13.0

### How do you run Kimai?

Virtual Server or alike

### Which PHP version are you using?

8.2

### Screenshots

![image](https://github.com/kimai/kimai/assets/42431686/5c39ebe9-295d-4522-b96b-2852532aec25)

**Comment by kevinpapst** [comment, marked as answer]
Missing PHP package, likely `pdo-mysql` - read https://www.kimai.org/documentation/fresh-ubuntu-22.html

**Reply by kerkhc** [reply]
thank you for the quick response - indeed, it was the missing php-mysql.
